bin/prebuild.js

```diff
@@ -0,0 +1,15 @@
+const chalk = require('chalk');
+const git = require('git-describe');
+const path = require('path');
+const fs = require('fs-extra');
+const { version: appVersion } = require('../package.json')
+
+const info = git.gitDescribeSync({ longSemver: true });
+const file = path.resolve(__dirname, '..', 'projects', 'app', 'src', 'version.json');
+
+fs.writeFileSync(
+  file,
+  `{ "version": "${appVersion}", "gitInfo": ${JSON.stringify(info)}}`,
```

Comment:
I think `JSON.stringify({ version: appVersion, gitInfo: info })` would be prettier here.